packages/web/src/components/swap/select-token-modal/SelectTokenModal.spec.tsx

```diff
@@ -0,0 +1,31 @@
+import SearchMenuModal from "./SelectTokenModal";
+import { render } from "@testing-library/react";
+import { Provider as JotaiProvider } from "jotai";
+import GnoswapThemeProvider from "@providers/gnoswap-theme-provider/GnoswapThemeProvider";
+import {
+  RecentdummyToken,
+  PopulardummyToken,
+} from "@containers/header-container/HeaderContainer";
+import {
+  coinList,
+  coinListProps,
+} from "@containers/swap-container/SwapContainer";
+
+describe("SearchMenuModal Component", () => {
+  it("should render", () => {
+    const mockProps = {
+      search: () => null,
+      keyword: "",
+      onSelectTokenModal: () => null,
+      coinList: coinList,
+      changeToken: (token: coinListProps, type: string) => null,
```

Comment:
The test succeeds even without the argument.

Reply:
deleted !